site/src/doc/quickstart.md

```diff
@@ -41,6 +41,10 @@ Open the `index.html` file in a text editor.  It should look something like this
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quick Start</title>
+    <style>
+      #map { height: 400px; }
```

Comment:
This conflicts with the rule in the style.css.  Not important to change, but I also don’t think it needs to be added.

Reply:
You are correct, I didn't notice the css imported from the js file.